x-pack/plugins/security_solution/common/api/search_strategy/network/top_n_flow.ts

```diff
@@ -21,5 +22,13 @@ export const networkTopNFlowSchema = requestOptionsPaginatedSchema.extend({
 });
 
 export type NetworkTopNFlowRequestOptionsInput = z.input<typeof networkTopNFlowSchema>;
-
 export type NetworkTopNFlowRequestOptions = z.infer<typeof networkTopNFlowSchema>;
+
+export const networkTopNFlowCountSchema = requestBasicOptionsSchema.extend({
+  ip: z.string().ip().nullable().optional(),
```

Comment:
NIT: `nullable()` and `optional()` together might read be a bit confusing:

`nullable()`  -  means `nullable: string | null;`
`optiona()` - means `optional?: string | undefined;`

So wdyt about using `.nullish()` that conveys the combined semantics a bit more explicitly?

```suggestion
  ip: z.string().ip().nullish(),
```